modules/misc/src/InternalVolume.C

```diff
@@ -6,11 +6,13 @@
 /****************************************************************/
 #include "InternalVolume.h"
 
+#include "MooseMesh.h"
+
 template <>
 InputParameters validParams<InternalVolume>()
 {
   InputParameters params = validParams<SideIntegralPostprocessor>();
-  params.addParam<unsigned int>("component", 1, "The component to use in the integration");
+  params.addRangeCheckedParam<unsigned int>("component", 1, "component<3", "The component to use in the integration");
```

Comment:
I'm actually not sure of that... I don't know the specifics of when / how this check gets applied.

Plus, this is more self-documentary...